modular_skyrat/modules/catnip/code/catnip_trauma.dm

```diff
@@ -0,0 +1,77 @@
+/datum/brain_trauma/special/laser_pointer
+	name = "laser pointers"
+	desc = "Patient seems to be a feline."
+	scan_desc = "feline tendencies"
+	gain_text = "<span class='warning'>LIGHT.</span>"
+	lose_text = "<span class='notice'>Where'd that light go?</span>"
```

Comment:
```suggestion
	span_warning("LIGHT.")
	span_notice("Where'd that light go?")
```

Reply:
Literally nothing uses them for gain and lose text, and I'm _somewhat_ sure there's a reason for it though I wouldn't be able to tell you what it was.